### PR TITLE
Edits to text in API Conventions docs

### PR DESCRIPTION
**Pull request by dmeiss**
Minor edits to phrasing, punctuation, capitalization, and formatting.

### CODE REVIEW

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

File: docs/reference/api-conventions.asciidoc [hunk 1]
```diff
@@ -526,7 +528,7 @@ If one of these quantities is large we'll print it out like 10m for 10,000,000 o
 when we mean 87 though. These are the supported multipliers:
 
 [horizontal]
-``::   Single
+`s`::  Single
```
Comment (danielmitterdorfer) [inline docs/reference/api-conventions.asciidoc]: The `s` suffix is not recognized by Elasticsearch (see also https://github.com/elastic/elasticsearch/blob/869de3f160a727fedf2eb8dc41713e9dce24ba5f/server/src/main/java/org/elasticsearch/common/unit/SizeValue.java#L170-L197) so this needs to be an empty string.

**Review by danielmitterdorfer — CHANGES_REQUESTED** [review]
Thanks for these fixes @dmeiss. The changes look mostly fine to me but I left one remark about an incorrect multiplier. Can you please remove the "s" multiplier?

**Comment by dmeiss** [comment]
Made requested changes. Have a question though about how that empty string should be represented. It’s currently `` but I’m wondering if it should be “” instead.

**Comment by danielmitterdorfer** [comment]
> Have a question though about how that empty string should be represented. It’s currently `` but I’m wondering if it should be “” instead.

On second thought we should just remove the entire line. I doubt that it helps anybody if we document that if no multiplier is specified, we treat the value as is. I think this is expected behavior anyway.

**Comment by dmeiss** [comment]
I agree. Would you like me to go ahead & remove it?

**Comment by dmeiss** [comment]
Went ahead & removed the "single" line

**Review by danielmitterdorfer — APPROVED** [review]
Thanks! Looks good to me.

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Pull request merged**